NExtends/Expressions/ExpressionHelper.cs

```diff
@@ -0,0 +1,236 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+
+namespace NExtends.Expressions
+{
+    public class ExpressionHelper
+    {
+        // Utile pour les types créés à partir de Group By d'un autre type
+        public static Expression<Func<TOut, bool>> Transfert<TIn, TOut>(Expression<Func<TIn, bool>> expression)
+        {
+            return TransfertWithRename<TIn, TOut>(expression, new Dictionary<string, string>());
+        }
+        /// <summary>
+        /// Permet de transférer un délégué d'un enfant vers son parent étant connu le nom de la propriété qui permet de passer du parent à l'enfant
+        /// </summary>
+        /// <typeparam name="TChild">Le type enfant</typeparam>
+        /// <typeparam name="TParent">Le type parent</typeparam>
+        /// <param name="expression">L'expression (MemberExpression en général) sur l'enfant</param>
+        /// <param name="childName">Le nom de la propriété permettant d'accéder à l'enfant depuis le parent</param>
+        /// <returns></returns>
+        public static LambdaExpression TransfertToParent<TChild, TParent>(LambdaExpression expression, string childName)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') 'TChild' is not used in the method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS2326)